doc/htmldoc/benchmark_results.rst

```diff
@@ -0,0 +1,90 @@
+.. _nest_benchmark_results:
+
+NEST performance benchmarks
+===========================
+
+
+NEST performance is continuously monitored and improved across various network sizes.
+Here we show benchmarking results for NEST version 3.8 on Jureca-DC.
+
+
+.. grid:: 1 1 2 2
+
+   .. grid-item::
+       :columns:  5
+       :class: sd-align-major-center
+
+       .. figure:: /static/img/replace_me.png
+
+           Wall time versus number of nodes
+
+           some more text
+
+   .. grid-item::
+       :columns:  7
+
+       **Strong scaling experiment of the Microcircuit model** [1]_
+
+       * The model has ~80 000 neurons and ~300 million synapses
+       * Increasing number of computing resources decrease simulation time
+       * The model runs faster than real time when using more than 2 nodes
+
+
+   .. grid-item::
+       :columns: 5
+       :class: sd-align-major-center
+
+       .. figure:: /static/img/replace_me.png
+
+           Wall time versus number of nodes
+
+   .. grid-item::
+       :columns: 7
+
+       **Strong scaling experiment of the Multi-area-model** [2]_
+
+       * The model has ~3.2 million neurons and ~10 billion synapses
+       * Steady decrease of run time with additional compute resources
+
+
+   .. grid-item::
+       :columns: 5
+       :class: sd-align-major-center
+
+       .. figure:: /static/img/replace_me.png
+
+         Wall time versus number of nodes
+
+   .. grid-item::
+       :columns: 7
+
+       **Weak scaling experiment of the HPC benchmark model** [3]_
+
+       * The size of network scales proportionally with the computational resources used
+       * Largest network size: ~5.6 million neurons and ~61 billion synapses
```

Comment:
Maybe more specifically
```suggestion
       * Largest network size in this diagram: ~5.6 million neurons and ~61 billion synapses
```
or `to date`, or `benchmarked`. This is not the "largest network size" in any general sense.